homeassistant/components/darksky/weather.py

```diff
@@ -67,10 +70,13 @@
         vol.Optional(CONF_MODE, default="hourly"): vol.In(FORECAST_MODE),
         vol.Optional(CONF_UNITS): vol.In(["auto", "si", "us", "ca", "uk", "uk2"]),
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(
+            CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL
```

Comment:
This option is already present in the base platform schema.

Reply:
@MartinHjelmare I mixed up CONFIG_SCHEMA definitions with PLATFORM_SCHEMA defintion.  I'll remove it.